scripts/convert_user_roles/convert_user_to_team_member_roles.py

```diff
@@ -0,0 +1,102 @@
+import os
+import requests
+
+OPSLEVEL_API_TOKEN = os.environ["OPSLEVEL_API_TOKEN"]
+OPSLEVEL_ENDPOINT = "https://app.opslevel.com/graphql"
+
+LIST_USERS_QUERY = """
```

Comment:
I see that the output of this query is shown to the user -- IMO it would be pretty useful to be able to see a list of what teams the user is part of. 

Tags might be potentially useful here too.